runtime/tutor.txt

```diff
@@ -416,6 +415,338 @@ _________________________________________________________________
 
 
 
+
+=================================================================
+=                   SELECTING TO A CHARACTER                    =
+=================================================================
+
+ Select up to, and including a character with f, followed by that
+ character. Select backwards to a character with F (uppercase).
+ The f stands for 'find'.
+
+ 1. Move the cursor to the line below marked -->. Make sure your
+    cursor is put just after the arrow.
+ 2. Press f, followed by the character you want to select
+    to, in this case a square bracket: [.
+ 3. press d to delete your selection, breaking the sentence free
+    Of the square brackets.
+ 4. Go to the very end of the line and repeat again with
+    uppercase F. Remember this time you're selecting to a
+    closing ].
+
+ --> -----[Free this sentence of it's brackets!]-----
+
+ Note: Unlike Vim, Helix doesn't limit this command to just one
+       line. It'll just search for the character in the file.
+=================================================================
+=                  SELECTING 'TIL A CHARACTER                   =
+=================================================================
+
+ Select up to, but NOT including ('til) a character with t.
+ Select backwards until a character with uppercase T.
+
+ 1. Move the cursor to the line below marked -->.
+ 2. Press t and then A to select until A.
+ 3. Press d to delete the dashes.
+ 4. Go to the line end, and repeat with T. Only
+    this time you're searching for an 's'.
+
+ --> -----A sentence surrounded in dashes-----
+
+ Note: Unlike Vim, Helix doesn't limit this command to just one
+       line. It'll just search for the character in the file.
```

Comment:
I think it might be better to merge this two sections or having the tutor too long may be overwhelming to new users.

Reply:
I have just done this but I had to shorten it somewhat to fit into a 80x24 screen. Trying to fit everything into that size is becoming an issue.